Source/Other/Formatters/SAUuidFormatter.swift

```diff
@@ -0,0 +1,180 @@
+//
+//  Created by Luis Aguiniga on 2024.07.07
+//  Copyright © 2024 Sequel-Ace. All rights reserved.
+//
+
```

Comment:
One little question from me - does this play nice with the other recent PR around UUIDs? https://github.com/Sequel-Ace/Sequel-Ace/pull/2051

Reply:
Unfortunately I don't have a MariaDB accessible at the moment so to  confirm but looks like #2051 deals specifically with typegrouping == UUID  (MariaDb only) and this PR is dealing specifically with binary(16) so I think we should be ok since with my code, the right click menu won't be accessible unless its the right typegroup & length.

Side note:  I just double checked the field type check in my code and I had a copy paste error, was checking for binary twice. Fixed to check for typegrouping == binary && length == 16 so that the above claim holds true.

Reply:
Ok I managed to get MariaDB installed in one of my VMs. There was a minor hiccup with my code preventing UUID from getting edited in the pop editor but latest commit fixes that. So I have confirmed this PR plays nice with #2051

Reply:
Awesome! Thank you!!